easy-connect.h

```diff
@@ -3,22 +3,24 @@
 
 #include "mbed.h"
 
-Serial output(USBTX, USBRX);
-
 #define ETHERNET        1
 #define WIFI_ESP8266    2
 #define MESH_LOWPAN_ND  3
 #define MESH_THREAD     4
+#define WIFI_ODIN       5
 
 #if MBED_CONF_APP_NETWORK_INTERFACE == WIFI_ESP8266
 #include "ESP8266Interface.h"
 
 #ifdef MBED_CONF_APP_ESP8266_DEBUG
-ESP8266Interface esp(MBED_CONF_APP_ESP8266_TX, MBED_CONF_APP_ESP8266_RX, MBED_CONF_APP_ESP8266_DEBUG);
+ESP8266Interface wifi(MBED_CONF_APP_WIFI_TX, MBED_CONF_APP_WIFI_RX, MBED_CONF_APP_ESP8266_DEBUG);
```

Comment:
Why have two? We don't have, nor have we ever had that kind of config in the mbed-os-example-client. It's simpler if we only use the WIFI TX/RX, that way there's only way to configure it. One is simpler than two.